int32 index
uint64 time_stamp
uint8[5242880] data

uint32 MAX_SIZE = 5242880
